### PR TITLE
Update Get-Commands.md

### PR DESCRIPTION
**Pull request by lebluxtv**
added /me to messages in the code example to keep the same formatting as the example above ( without code )

### CODE REVIEW

**Review by Ik1497 — COMMENTED** [review]
I think it should probably be changed to CPH.SendAction instead, so when you use the YouTube one it won't give the twitch bit at the front
image

File: streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md [hunk 1]
```diff
@@ -66,15 +66,15 @@ public class CPHInline
     public bool Execute()
     {
         List<string> commands = (List<string>)args["commandsList"];//This is when the Variable Name in the Get Commands Sub-Action is `commands`
-        string message = "Commands Avaliable to you: ";
+        string message = "/me Commands Avaliable to you: ";
```
Comment (GoWMan813) [inline streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md:69]: Should remove the /me

File: streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md [hunk 2]
```diff
@@ -66,15 +66,15 @@ public class CPHInline
...
         bool bot = false;
         int messageMax = 495; //Limit is 500 on Twitch, 200 on YouTube.. I took a few off as a safety measure.
         for (int i = 0; i < commands.Count; i++)
         {
             if ((commands[i].Length + message.Length) > messageMax)
             {
                 CPH.SendMessage(message, bot); // This will need to be changed to CPH.SendYouTubeMessage(message, bot); if you wish to send the message to Youtube
```
Comment (GoWMan813) [inline streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md:76]: Should be CPH.SendAction for Twitch.

File: streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md [hunk 3]
```diff
@@ -66,15 +66,15 @@ public class CPHInline
...
-                message = "Addional Commands for you: ";
+                message = "/me Addional Commands for you: ";
```
Comment (GoWMan813) [inline streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md:77]: Should remove the /me.

**Review by GoWMan813 — CHANGES_REQUESTED** [review]
Slash commands should not be part of docs as slash commands have been deprecated.  While Twitch does still allow use of /me it is highly advisable to use supplied CPH.SendAction Method to not confuse users of slash command abilities.

File: streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md [hunk 4]
```diff
@@ -88,4 +88,4 @@ public class CPHInline
 ---
 
 - [<i class="mdi mdi-chevron-left"></i> **Commands Sub-Actions Reference *Go Back***](/Sub-Actions/Commands)
```
Comment (GoWMan813) [inline streamerbot/3.api/1.sub-actions/core/commands/Get-Commands.md:90]: This whole section can be removed as well.  Artifacts left over from the old wiki system.
[<i class="mdi mdi-chevron-left"></i> **Commands Sub-Actions Reference *Go Back***](/Sub-Actions/Commands)
{.btn-grid .my-5}

**Pull request closed without merging**